src/template/output.ts

```diff
@@ -16,6 +16,10 @@ export class Output extends TemplateImpl<OutputToken> implements Template {
   }
   public * render (ctx: Context, emitter: Emitter) {
     const val = yield this.value.value(ctx)
-    emitter.write(stringify(toValue(val)))
+    if (ctx.opts.keepOutputType) {
+      emitter.write(toValue(val))
+    } else {
+      emitter.write(stringify(toValue(val)))
```

Comment:
We should return the `val` as it is, and always treat it inside emitters.

> BTY, I see `stringify` already called `toValue`, see https://github.com/harttle/liquidjs/blob/236d97170519337dcb216e228d93b9208a23081e/src/util/underscore.ts#L32

Reply:
Since I am really new to liquidjs code base, I tried to make as few code change as possible in order to accomplish the output I desired.

I am however happy to refactor some of the logic and put the existing logic inside the emitter.write function

Reply:
Addressed in commit 85f30fd9dce27914357b0b547ea1cbd6bd988962